.github/workflows/docs.yml

```diff
@@ -113,3 +113,20 @@ jobs:
       - name: Deploy with mike 🚀
         run: |
           pixi run deploy-dev
+
+      - name: Setup Git
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+
+      - name: Copy file to gh-pages
+        run: |
+          git fetch
+          git checkout gh-pages
+          git checkout main -- install/install.sh install/install.ps1
```

Comment:
I think it would be great to only do thsi fi the files have chagned to reduce the number of commits.

Reply:
Now I'm only committing if there is a diff